.github/workflows/ci.yml

```diff
@@ -12,12 +12,12 @@ jobs:
       # To use this repository's private action,
       # you must check out the repository
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Setup NodeJS
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: 16
```

Comment:
can you change this to read the version from the package.json file then update the package.json file to have the `engines` property which specifies the node version (20 in this case)?

Reply:
https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

Reply:
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines